vignettes/installation.Rmd

```diff
@@ -17,15 +17,11 @@ As the `azureml` package is not available on CRAN, you will need `remotes` to in
 ``` {r Install Remotes, eval=FALSE}
 install.packages('remotes')
 ``` 
-Then, you can use the `install_github` function to install the package. Until the current repo is opened to the public, you will need to use a [generated personal access token](https://github.com/settings/tokens). When generating the token, make sure to select the "repo" scope.
+Then, you can use the `install_github` function to install the package.
 
 ``` {r Install Azureml, eval=FALSE}
-remotes::install_github('https://github.com/Azure/azureml-sdk-for-r',
-                        auth_token = '<your personal access token>',
-                        INSTALL_opts=c("--no-multiarch"),
-                        build_vignettes = FALSE)
+remotes::install_github('https://github.com/Azure/azureml-sdk-for-r')
 ```
-Set `build_vignettes` to TRUE if you would like the vignettes to be downloaded along with the main package.
```

Comment:
Are we changing to have vignettes automatically be downloaded? Otherwise, I think we should keep this line and the parameter in the example above.

Reply:
discussed with Billy, it's okay to have vignettes downloaded with the package

Reply:
@diondrapeck , the idea is to keep the installation instruction as simple as possible. as long as it works for most of people, we are good. For special cases, mention in the troubleshooting section or notes.

Reply:
sounds good!